recipes/cuda-gdb/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+
+for i in `ls`; do
+    [[ $i == "build_env_setup.sh" ]] && continue
+    [[ $i == "conda_build.sh" ]] && continue
+    [[ $i == "metadata_conda_debug.yaml" ]] && continue
```

Comment:
Are these still relevant?

Reply:
Yes, due to the lazy copying we do in these scripts. Tested without and they are picked up:
```
‘build_env_setup.sh’ -> ‘$PREFIX/build_env_setup.sh’
‘conda_build.sh’ -> ‘$PREFIX/conda_build.sh’
‘metadata_conda_debug.yaml’ -> ‘$PREFIX/metadata_conda_debug.yaml’
```